index.html

```diff
@@ -0,0 +1,129 @@
+            <ul class="navbar-nav pff-regular">
+              <li><img class="nav-item close-icon resp-hidden" src="img/close-icon.png" alt="close tag"></li>
+              <li class="nav-item navbar-margin">
+                <a class="nav-link active" aria-current="page" href="about.html">About</a>
+              </li>
+              <li class="nav-item navbar-margin">
+                <a class="nav-link" href="index.html">Home</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </div>
+    <div class="container-fluid d-flex flex-column header-landing" id="header-landing">
+        <p class="pff-light col-12 col-md-9 s-color" id="p-hello">"Hello! We make you RESPONSIVE..."</p>
+        <h1 class="pff-bold col-12 col-md-9">THE BEST WEB PAGE BUILDERS OF THE YEAR</h1>
+        <p class="sff-regular col-12 col-md-8 p-color" id="p-header">
+            Contrary to popular belief, Lorem Ipsum is not simply random text. 
+            It has roots in a piece of classical Latin literature from BC, 
+            making it over years old. Richard McClintock, a Latin professor 
+            at Hampden-Sydney College in Virginia.
+        </p>
+        <p class="pff-bold" id="p-deal">Get our SUPER DEALS of March!...</p>
+        <p class="sff-regular" id="p-opo">@ Our Personal Org. in the World Wide Web and more</p>
+    </div>
+  </header>
+  <section class="container-fluid" id="main-sec">
+    <div class="d-flex flex-column">
+      <h2 class="t-color main-h2">Main Tegnologies</h2>
+      <hr class="s-color"/>      
+      <div class="d-flex flex-column" id="main-container">
+        <div class="d-flex flex-row col-12 col-md-2 main-card">
+          <img class="main-icon" src="img/conference-icon.png" alt="conference icon"/>
+          <h3 class="s-color pff-bold main-h3">Forum</h3>
+          <p class="t-color sff-bold p-main">
+            Contrary to popular belief, Lorem Ipsum is not simply random text. 
+            It has roots in a piece of classical Latin literature from BC, 
+          </p>
+        </div>
+        <div class="d-flex flex-row col-12 col-md-2 main-card">
+          <img class="main-icon" src="img/expositon-icon.png" alt="expositon icon"/>
+          <h3 class="s-color pff-bold main-h3">Forum</h3>
+          <p class="t-color sff-bold p-main">
+            Contrary to popular belief, Lorem Ipsum is not simply random text. 
+            It has roots in a piece of classical Latin literature from BC, 
+          </p>
+        </div>
+        <div class="d-flex flex-row col-12 col-md-2 main-card">
+          <img class="main-icon" src="img/taller-icon.png" alt="taller icon"/>
+          <h3 class="s-color pff-bold main-h3">Forum</h3>
+          <p class="t-color sff-bold p-main">
+            Contrary to popular belief, Lorem Ipsum is not simply random text. 
+            It has roots in a piece of classical Latin literature from BC, 
+          </p>
+        </div>
```

Comment:
[OPTIONAL]
- I would recommend fixing the size of the images on your website so the images are not stretched.
<img width="206" alt="image" src="https://user-images.githubusercontent.com/77190203/158040142-317da012-c553-44f1-a9ef-994979b0b6ca.png">

Reply:
The image from my browser's inspector and github pages
![image](https://user-images.githubusercontent.com/98621809/158041051-03ed6399-d672-4cff-a288-718f72a2cbf9.png)